modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/HelpPage.kt

```diff
@@ -0,0 +1,541 @@
+                        ContainerState.Error
+                    }
+
+                    loadingState !is LoadingState.Finished && !loadedBefore -> ContainerState.Loading
+                    else -> {
+                        loadedBefore = true
+                        ContainerState.Loaded
+                    }
+                }
+            }
+        }
+
+        val animatedAlpha by animateFloatAsState(
+            targetValue = if (containerState == ContainerState.Loaded) 1f else 0f,
+        )
+        // Do not animate alpha if there is an error state to avoid showing web error content
+        val alpha = if (containerState == ContainerState.Error) 0f else animatedAlpha
+        HelpWebView(
+            state = webViewState,
+            onSendFeedbackEmail = onSendFeedbackEmail,
+            onContactSupport = onContactSupport,
+            onTapUri = onTapUri,
+            onLoadUrl = { initialUrl = it },
+            onWebViewCreated = onWebViewCreated,
+            onWebViewDisposed = onWebViewDisposed,
+            modifier = Modifier
+                .fillMaxSize()
+                .alpha(alpha),
+        )
+        AnimatedContent(
+            targetState = containerState,
+        ) { state ->
+            when (state) {
+                ContainerState.Loading -> WebViewProgressIndicator(
+                    modifier = Modifier.fillMaxSize(),
+                )
+
+                ContainerState.Error -> WebViewError(
+                    onContactSupport = onContactSupport,
+                    modifier = Modifier.fillMaxSize(),
+                )
+                // Web View is not part of AnimatedContent because we want to keep it active
+                // to load pages regardless of the container state
+                ContainerState.Loaded, ContainerState.Initalizing -> Unit
+            }
+        }
+    }
+}
+
+private enum class ContainerState {
+    Initalizing,
+    Loading,
+    Loaded,
+    Error,
+}
+
+@Composable
+private fun WebViewProgressIndicator(
+    modifier: Modifier = Modifier,
+) {
+    Box(
+        contentAlignment = Alignment.Center,
+        modifier = modifier,
+    ) {
+        CircularProgressIndicator(
+            color = MaterialTheme.theme.colors.secondaryIcon01,
+            modifier = Modifier.size(36.dp),
+        )
+    }
+}
+
+@Composable
+private fun WebViewError(
+    onContactSupport: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        verticalArrangement = Arrangement.Center,
+        modifier = modifier.padding(horizontal = 48.dp),
+    ) {
+        TextP40(
+            text = stringResource(LR.string.settings_help_cant_load),
+            textAlign = TextAlign.Center,
+        )
+        Spacer(
+            modifier = Modifier.height(8.dp),
+        )
+        RowButton(
+            text = stringResource(LR.string.settings_help_contact_support),
+            onClick = onContactSupport,
+        )
+    }
+}
```

Comment:
Could you center the text for landscape as well? 

![Screenshot_20250114_115644](https://github.com/user-attachments/assets/63576bbe-3164-40b3-9d2c-c66315569651)

Reply:
Fixed in b909004ac920ceb6e7bfb42d0f2e2f9abf095112